backend/src/main/java/io/diveni/backend/controller/WebsocketController.java

```diff
@@ -232,4 +236,16 @@ public synchronized void adminSelectedUserStory(
     webSocketService.sendSelectedUserStoryToMembers(session, index);
     LOGGER.debug("<-- adminSelectedUserStory()");
   }
+
+  @GetMapping("/get-timer-value")
+  public synchronized ResponseEntity<Long> getTimeValue(String memberID) throws ParseException {
+    LOGGER.debug("--> get-timer-value");
+    Session session =
+        ControllerUtils.getSessionByMemberIDOrThrowResponse(databaseService, memberID);
+    Date startDate = Utils.getDateFromString(session.getTimerTimestamp());
+    Date currentDate = new Date();
+    long timeDifference = currentDate.getTime() - startDate.getTime();
+    LOGGER.debug("<-- get-timer-value");
+    return new ResponseEntity<>(timeDifference, HttpStatus.CREATED);
```

Comment:
Why status `CREATED`? Should it not be `OK`?

Reply:
Yeah you're right. Thanks I didnt notice it.